src/visitors/semantic/mod.rs

```diff
@@ -195,20 +194,15 @@ pub struct ImportFinderVisitor {
 impl<'a> Visitor<'a> for ImportFinderVisitor {
     fn visit(&mut self, node: Node<'a>) -> bool {
         if let Node::ImportDeclaration(import) = node {
-            let alias = match import.alias.clone() {
+            let name = match import.alias.clone() {
                 Some(alias) => alias.name.to_string(),
                 None => get_package_name(import.path.value.as_str())
-                    .unwrap_or_default()
                     .to_owned(),
             };
 
             self.imports.push(Import {
                 path: import.path.value.clone(),
-                alias,
-                initial_name: get_package_name(
-                    import.path.value.as_str(),
-                )
-                .map(|s| s.to_owned()),
```

Comment:
Are we replacing two entries `alias` and `initial_name` with one entry `name`? Just wondering why?

Reply:
`alias` was never used, but something in the way the linter was looking at it, and I just changed `initial_name` to `name`